### DISCUSSION TITLE
HI, Maui Project Port

### DISCUSSION THREAD

**Discussion by lukewire129** [JamesnetGroup/navigationbar#3] [Show and tell]
I couldn't run the iPhone because I didn't have a Mac or developer account,
I've checked that Windows and Android are working fine
(Oh of course, Android might not be seen on some devices because of its size!)

![KakaoTalk_20240113_193330953](https://github.com/vickyqu115/navigationbar/assets/54387261/edc6bc42-e49c-4339-aefc-409758044987)

https://github.com/lukewire129/MauiNavigationBar

**Comment by vickyqu115** [comment]
Wow～I haven't tried cross-platform development other than WPF, but the idea of implementing it through XAML and C# is really interesting and amazing!!

**Reply by lukewire129** [reply]
I also took this opportunity to learn
It was interesting and exciting to be able to implement a listbox that didn't exist on maui.
